libs/barista-components/alert/src/alert.ts

```diff
@@ -29,12 +30,27 @@ import {
     role: 'alert',
     class: 'dt-alert',
     '[class.dt-alert-warning]': 'severity === "warning"',
+    '[class.dt-alert-info]': 'severity === "info"',
+    '[class.dt-alert-success]': 'severity === "success"',
   },
   preserveWhitespaces: false,
   changeDetection: ChangeDetectionStrategy.OnPush,
   encapsulation: ViewEncapsulation.Emulated,
 })
 export class DtAlert {
   /** The severity type of the alert. */
-  @Input() severity: 'error' | 'warning' = 'error';
+  @Input() severity: 'error' | 'warning' | 'info' | 'success' = 'error';
```

Comment:
Don't we use Object literals for types like this?

Reply:
Did you mean TypeScript types?
```
type AlertSeverityType = 'error' | 'warning' | 'info' | 'success';
```

Reply:
ie string enums
```ts
enum Severity {
  ERROR = 'error',
  WARNING = 'warning',
}
```

Reply:
No, we use tuple types without a separate type as it is more consumer-friendly. When you hide the values in a `type` the consumer will get the `type AlertServityType` in his autocompletion. If you provide the tuple directly the consumer has first-class autocompletion with the appropriate values.